packages/client/src/components/ClientProvider/ClientProvider.tsx

```diff
@@ -161,7 +162,13 @@ const ClientProvider = ({
               error: CombinedError,
               operation: Operation<unknown, AnyVariables>,
             ) => {
-              const errorMessages = extractErrorMessages(error);
+              let errorMessages = extractErrorMessages(error);
+
+              const validationMessageKeys = extractValidationMessageKeys(error);
+              if (validationMessageKeys) {
+                errorMessages = validationMessageKeys;
```

Comment:
This will overwrite error messages from the first function, right?  Why not get the results of both and concatenate them?  Will there be duplicates?

Reply:
Yeah, I did that intentionally. Only overrides if there are validation error enums. 
Sometimes it isn't a very useful message (Validation failed for the field ...) and sometimes it duplicates. 
For cases like authorization problems, there won't be a validation messages array so it returns the first function. 

If I concatenate  
`error.message` is returning what is in the array `error.extensions.validation`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/d0c4ca26-90a6-48d6-bc99-3e639906054b)